test/resources/Applicants.test.ts

```diff
@@ -19,17 +19,30 @@ function sort_by_firstname( a: Applicant, b: Applicant ) {
   return 0;
 }
 
-let modifiedApplicant = { ... exampleApplicant,
-                          firstName: "Test2" }
-
 let applicant: Applicant;
 
+async function init() {
+  applicant = await createApplicant();
+}
+
+beforeEach(() => {
+  return init();
```

Comment:
Actual question. Do we need the `return` here? 😮

Reply:
It looks so, otherwise tests fails. I believe that's due to the fact that `init` function is declared as `async`, therefore if we don't return the function, the tests are proceed without waiting for the async initialization to be completed.

Reply:
As `init()` function is now collapsed into the anonymous function which was calling it, this thread is now meaningless.